src/components/Common/Title.tsx

```diff
@@ -0,0 +1,22 @@
+import { memo } from "react";
+import styled from "styled-components"
```

Comment:
A semicolon is missed. P.s. why prettier didn't auto-fix that?

Reply:
I didn't run format